src/main/java/com/google/sps/servlets/CalendarServlet.java

```diff
@@ -43,11 +43,13 @@
  * Servlet responsible for returning the link to the shared family calendar
 */
 public class CalendarServlet extends AbstractAppEngineAuthorizationCodeServlet {
+
+  static final String APP_NAME = "Household";
```

Comment:
Is this variable used anywhere?

Reply:
Oops I think it's used in a different area, will remove